Makefile

```diff
@@ -27,15 +27,20 @@ endif
 ##@ Default target
 all: gitops ## Install dependencies and build Gitops binary
 
+
+TEST_TO_RUN=./...
 ##@ Test
-unit-tests: dependencies  ## Run unit tests
+unit-tests: dependencies ## Run unit tests
+	which ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
When I run this locally, I always get the output:

```
which ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo
/Users/david/go/bin/ginkgo
```

Given this is going to run on every test command, is there something we can do to make it quieter, at least in the case where Ginkgo is already installed?

Reply:
I see what you did here: using `@go install github.com/onsi/ginkgo/v2/ginkgo` and letting Go quietly handle the case where it’s already installed. Nice. 👍